ci/forms.py

```diff
@@ -142,14 +143,13 @@ class ReportGenerationForm(forms.Form):
     output_file = forms.CharField(
         widget=forms.TextInput(attrs={'placeholder': 'Name of output file'}),
         required=True)
-    start_date = forms.CharField(
-        widget=forms.TextInput(attrs={'placeholder': 'YYYY-MM-DD'}),
+    start_date = forms.DateField(
+        widget=DatePicker(options={'placeholder': 'YYYY/MM/DD','format': 'yyyy/mm/dd','autoclose': True}),
```

Comment:
Did you check that this format works with the API? I suspect it needs to receive the date as `YYYY-MM-DD`